components/o-message/src/tsx/message.tsx

```diff
@@ -0,0 +1,84 @@
+import { Fragment } from 'react';
+
+type MessageContent = {
+	detail: string;
+	hightLight?: string;
+	additionalInfo?: string;
+}
+
+type MessageAction = {
+	copy: string;
+	url: string;
+	openInNewWindow?: boolean
+}
```

Comment:
```suggestion
type MessageAction = {
	text: string;
	url: string;
	openInNewWindow?: boolean
}
```

actually, let's make it `text` here to match [the javascript api of this component ](https://registry.origami.ft.com/components/o-message@5.2.2/readme?brand=core#options)